modules/accounts.js

```diff
@@ -206,46 +211,6 @@ Accounts.prototype.isLoaded = function () {
  * @see {@link http://apidocjs.com/}
```

Comment:
I think finally as a part of this PR the todo mentioned line above should be done:
` * @todo implement API comments with apidoc.`